opencti-platform/opencti-front/src/utils/Localization.js

```diff
@@ -4520,12 +4508,6 @@ const i18n = {
       'infrastructure-operator': 'インフラストラクチャ運用者',
       'malware-author': 'マルウェア作成者',
       sponsor: 'スポンサー',
-      status_NEW: '新規',
-      status_IN_PROGRESS: '進行中',
-      status_ANALYZED: '解析完了',
-      status_CLOSED: '終了済',
-      status_PENDING: '保留',
-      status_TO_BE_QUALIFIED: '取り掛かる',
```

Comment:
Just for my knowledge, we use Google Translate for these languages ?

Reply:
Yes :) Then we will let native people to fix it if necessary.